### PR TITLE
Change the layout of the minimap

### PR DESCRIPTION
**Pull request by jmdejong**
minimaplayout2
Since the minimap is now rotated, the layout of the minimap area should be somewhat different too.

### CODE REVIEW

File: horizons/gui/widgets/minimap.py [hunk 1]
```diff
@@ -279,18 +281,64 @@ def update_cam(self):
...
-		for i in range(0, 4):
-			self.minimap_image.rendertarget.addLine(self._get_render_name("cam"),
-			                                        minimap_corners_as_point[i],
-			                                        minimap_corners_as_point[(i + 1) % 4],
-			                                                         *self.COLORS["cam"])
+		xmin, ymin = self.transform.world_to_minimap(displayed_area[0])
+		xmax, ymax = self.transform.world_to_minimap(displayed_area[2])
+
+		# draw horizontal lines
+		for y in [ymin, ymax]:
```
Comment (AndyMender) [inline horizons/gui/widgets/minimap.py]: Tuples are lighter than lists so it's slightly better to use a tuple here:
```python
for y in (ymin, ymax):
```

File: horizons/gui/widgets/minimap.py [hunk 2]
```diff
@@ -279,18 +281,64 @@ def update_cam(self):
...
+			x1 = max(xmin, self.transform.get_min_x(y))
+			x2 = min(xmax, self.transform.get_max_x(y))
+			if x2 > x1:
+				drawLine(fife.Point(x1, y), fife.Point(x2, y))
+
+		# draw vertical lines
+		for x in [xmin, xmax]:
```
Comment (AndyMender) [inline horizons/gui/widgets/minimap.py]: Same as for the earlier comment.

File: horizons/gui/widgets/minimap.py [hunk 3]
```diff
@@ -753,14 +760,32 @@ def iter_points(self):
 				x = x_i + location.left
 				# TODO: better name
 				asdf = int(abs(x / location.width - 0.5) * location.height)
-				for y_i in range(asdf, location.height - asdf):
+				for y_i in range(self.get_min_y(x), self.get_max_y(x)):
```
Comment (AndyMender) [inline horizons/gui/widgets/minimap.py]: Good change. The earlier code wasn't very clear. You can remove `asdf ` entirely now, I think.

Comment (jmdejong) [reply horizons/gui/widgets/minimap.py]: You're right! I didn't even realize.

File: horizons/gui/widgets/minimap.py (hunk 1 above)
Comment (jmdejong) [reply horizons/gui/widgets/minimap.py]: I thought I deleted this code in a later commit?

Comment (AndyMender) [reply horizons/gui/widgets/minimap.py]: My bad, yes, you did.

**Pull request merged**